designs/2018-simplified-package-loading/README.md

```diff
@@ -0,0 +1,87 @@
+- Start Date: 2018-12-27
+- RFC PR: (leave this empty, to be filled in later)
```

Comment:
Just a reminder to fill this in with https://github.com/eslint/rfcs/pulls/7 before merging.

Reply:
Updated.